pkg/server/app/app.go

```diff
@@ -363,6 +364,30 @@ func (ops *AppOperations) List(user *database.User) ([]*AppListItem, error) {
 	return items, nil
 }
 
+func (ops *AppOperations) SetAutoScale(user *database.User, appName string, as *AutoScale) error {
+	app, err := ops.checkPermAndGet(user, appName)
+	if err != nil {
+		return err
+	}
+
+	if err := ops.saveApp(app, user.Email); err != nil {
+		return teresa_errors.NewInternalServerError(err)
+	}
```

Comment:
Ok, LGTM but sounds a bit weird, I understand that it's necessary but still weird (change namespace annotation and autoScale).

I'm waiting for @aguerra opinion.